agent/xds/listeners.go

```diff
@@ -375,34 +379,16 @@ func makeListener(name, addr string, port int) *envoy.Listener {
 // any config generated by other systems will likely be in canonical protobuf
 // from rather than our slight variant in JSON/hcl.
 func makeListenerFromUserConfig(configJSON string) (*envoy.Listener, error) {
```

Comment:
It's typed configs everywhere now. This is a prerequisite for xDS v3, which is only typed.